### DISCUSSION TITLE
How hard is it to create a C64 SuperCPU

### DISCUSSION THREAD

**Discussion by banman11** [frntc/Sidekick64#28] [Ideas]
Just noticed this interesting thread on a C64 forum site.

https://www.melon64.com/forum/viewtopic.php?t=211

There seems to be much interest in CMD's SuperCPU.

How hard is it to implement this in the Sidekick64 cartridge?

**Comment by frntc** [comment]
Thanks for the link, haven't seen this one! I do have ideas how to implement CPU accelerators, but a straightforward implementation of a SuperCPU emulation is not possible. The SCPU makes use of DMA-transfers (to quickly copy memory from/to the C64 RAM) which are not possible with the Sidekick64.

**Comment by banman11** [comment]
No matter regarding the Super CPU. 

I am not very technical unfortunately. From what I can gather all that's really needed is an implementation 65c02 or 65816 then it will be useful according to the programmer's over at melon64.

I think definitely push out something. Even if it's only beta and has bugs. There is much interest in your project. Keep us posted....

The beauty of your platform is that if something needs changes it can all be accomplished via a software update.

Please feel free to join in at melon64. Your thoughts will be most welcome! 😁
